src/components/ContactForm/ContactForm.jsx

```diff
@@ -37,12 +37,12 @@ export default function ContactForm() {
                                             <div className="flex items-center mt-4 hidden md:flex ">
                                             <div className="flex-shrink-0 md:mx-20 mx-auto justify-center md:text-left md:justify-start ">
                                                 <div>
-                                                     <a href="/" >
+                                                     <a href="tel:+234081-1236-4568" >
                                                          <FontAwesomeIcon icon="phone"  className="hover:text-green "/><span >+234 081-1236-4568</span>
                                                     </a>
                                                 </div>
                                                 <div>
-                                                    <a href="/" >
+                                                    <a href="mailto:contact@Dilmun.com" >
```

Comment:
@Sima-J 

href attr. should be in curly braces 

check this for more details https://stackoverflow.com/questions/34779642/dynamic-href-tag-react-in-jsx/34779681

Reply:
@ismailkhalil20 I already updated and changed the codes the test, Every time the testing pass locally.